GCRCatalogs/catalog_configs/dc2_object_run2.2i_dr6a_matched.yaml

```diff
@@ -0,0 +1,16 @@
+#  Use ^/ to indicate file path relative to GCR root dir
+
+subclass_name: dc2_matched_table.DC2MatchedTable
+table_dir: /global/cscratch1/sd/jsanch87/DC2_Run2.2i/
```

Comment:
Update the directory path once https://github.com/LSSTDESC/desc-help/issues/28 is done.